recipe/meta.yaml

```
@@ -1,23 +1,26 @@
-{% set version = "1.5.0" %}
+{% set name = "pyevtk" %}
+{% set version = "1.6.0" %}
 
 package:
-  name: pyevtk
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/pyscience-projects/pyevtk/archive/v{{ version }}.tar.gz
-  sha256: 72fe9a4c9b55349a81fa0c424c4f7daaf7bea84814c178b12c38db17e0b385d6
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1f6be7876a3a005c8258861551da4fe7e44ff1a2e7ff2a93d6dc51deedfda5f4
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv
+  script: {{ PYTHON }} -m pip install . -vv
```

```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps
```
Although `grayskull` doesn't add it for some reason, I have run into trouble recently where a feedstock without `--no-deps` was bringing in the PyPI versions of dependencies and including them in the conda package.  Better to keep that flag.